### ISSUE TITLE
Question: how to resolve `ambiguous associated item` error?

### ISSUE THREAD

**Issue by kwonoj** [rkyv/bytecheck#20]
I have example code like below:

```
use rkyv::{Archive, Deserialize, Serialize};

#[derive(Archive, Deserialize, Serialize)]
#[archive_attr(repr(u32), derive(bytecheck::CheckBytes))]
enum LogLevel {
    Trace,
    Debug,
    Info,
    Warn,
    Error,
}
```

and checkbyte makes raise error for

```
error: ambiguous associated item
   --> src/lib.rs:5:34
    |
5   | #[archive_attr(repr(u32), derive(bytecheck::CheckBytes))]
    |                                  ^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[deny(ambiguous_associated_items)]` on by default
    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
    = note: for more information, see issue #57644 <https://github.com/rust-lang/rust/issues/57644>
note: `Error` could refer to the variant defined here
   --> src/lib.rs:11:5
    |
11  |     Error,
    |     ^^^^^
note: `Error` could also refer to the associated type defined here
   --> /Users/ojkwon/.cargo/registry/src/github.com-1ecc6299db9ec823/bytecheck-0.6.8/src/lib.rs:209:5
    |
209 |     type Error: Error + 'static;
```

Probably this can be suppressed by disabling rules, but curious if it's safe to do so and if there's a recommended way to avoid instead.

**Comment by djkoloski** [comment]
This looks like an issue caused by the proc macro not using fully qualified paths (like `Error` instead of `::bytecheck::Error`). I'll take a look at this.

**Comment by djkoloski** [comment]
Fixed by f2c4eff7570aa1002793a8efd6234f98b4c6424e
